support/hooks.js

```diff
@@ -1,4 +1,4 @@
-const {client} = require('nightwatch-cucumber');
+/*const {client} = require('nightwatch-cucumber');
```

Comment:
Is this whole file supposed to be commented out? If so, I'd be interested to know why 🙂